src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -35,13 +35,12 @@ public static partial class PlatformDetection
         public static bool IsOSX => RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
         public static bool IsNotOSX => !IsOSX;
         public static Version OSXVersion => IsOSX ?
-            ToVersion(Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment.OperatingSystemVersion) :
+            Environment.OSVersion.Version :
             throw new PlatformNotSupportedException();
```

Comment:
Do we still need OSXVersion at all?  Wondering if we can just use Environment.OSVersion.Version instead directly.

Reply:
No, we don't. Removed.